src/Commands/TraceCommand.php

```diff
@@ -98,16 +98,14 @@ private function appClasses()
 
     private function loadModel(string $class)
     {
-        if (!class_exists($class)) {
+        if (!class_exists($reflectionClass = new \ReflectionClass($class))) {
```

Comment:
Does this solve it? I thought `ReflectionClass` would throw an exception...

Reply:
no, it was just suppressing it.